templates/config.yaml

```diff
@@ -652,6 +652,11 @@ templates:
         values:
           <<: *go_kyma_component_1_11
           path: tests/asset-store
+      - to: ../prow/jobs/kyma/tests/rafter/rafter-test.yaml
```

Comment:
`rafter-test` and not `rafter`, because we already have `rafter.yaml` in test-infra for rafter repo, and prow does not allow for two such files